src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -632,6 +633,38 @@ default SplitsMode splitsOverlay()
 		return SplitsMode.OFF;
 	}
 
+	@ConfigSection(
+		name = "Path of Apmeken",
+		description = "Options for the Path of Apmeken.",
+		position = 900,
+		closedByDefault = true
+	)
+	String SECTION_APMEKEN = "sectionApmeken";
```

Comment:
Place below Path of Het (and below Akkha, if that merges first)

Reply:
Will reorder if/when the config PR is merged